Game/world/objects/npc.cpp

```diff
@@ -342,6 +342,11 @@ bool Npc::setPosition(const Tempest::Vec3& pos) {
   return setPosition(pos.x,pos.y,pos.z);
   }
 
+Tempest::Vec3 Npc::getPosition() const
```

Comment:
Duplicate of `Npc::position()`

Reply:
Good point, probably missed it because Npc::position() has auto as return type in the .h file.
Will remove the commit introducing the new function.

Reply:
One more question, why use the -> syntax in the .h file with auto return type?
Wouldn't it be easier to just write `Tempest::Vec3 position() const;`?
Or is it possible that other types are returned here?

And sorry if this is a stupid question, Its the first time that I see that syntax.